### PR TITLE
Create CatalogAPI for Space

### PR DESCRIPTION
**Pull request by panuhorsmalahti**

### CODE REVIEW

**Review by chenhunghan — APPROVED** [review]

**Comment by chenhunghan** [comment]
Wonder why there are linter errors..

**Comment by chenhunghan** [comment]
May eslint configs got updated?

**Comment by panuhorsmalahti** [comment]
Yeah, I added now lint fix command and fixed the issues. No idea why the PR that updated eslint didn't fail though.

File: src/LensPlatformClient.ts [hunk 1]
```diff
@@ -154,9 +154,9 @@ class LensPlatformClient {
    */
   get fetch() {
     const getToken = this.getToken.bind(this);
-    const defaultHeaders = this.defaultHeaders;
-    const httpAdapter = this.httpAdapter;
-    const logger = this.logger;
+    const { defaultHeaders } = this;
```
Comment (jweak) [inline src/LensPlatformClient.ts:157]: you could do `const { defaultHeaders, httpAdapter, logger } = this;`

Comment (panuhorsmalahti) [reply src/LensPlatformClient.ts:157]: True.. I think this was eslint's change

**Pull request merged**